frontend/src/components/pages/PeopleListingSection.tsx

```diff
@@ -1,130 +1,144 @@
+  //     referrer_info: {
+  //       name: "Hello",
+  //       twitter_url: "google.com",
+  //       twitter_photo_url:
+  //         "https://pbs.twimg.com/profile_images/1387824030602780673/CqiWzrma_400x400.jpg",
+  //     },
+  //   },
+  //   {
+  //     post_datetime: adj_date(10).toString(),
+  //     contact: {
+  //       twitter_handle: "mytwitter",
+  //       twitter_url: "google.com",
+  //     },
+  //     name: "bob",
+  //     twitter_photo_url:
+  //       "https://pbs.twimg.com/profile_images/1387824030602780673/CqiWzrma_400x400.jpg",
+  //     description:
+  //       "A description. Lorem ipsum dolor amit. I like pizza, pizza is good to eat. Please givem e more pizza",
+  //     lease_preference: "A day",
+  //     lease_timing: "soon",
+  //     referrer_info: {
+  //       name: "Hello",
+  //       twitter_url: "google.com",
+  //       twitter_photo_url:
+  //         "https://pbs.twimg.com/profile_images/1387824030602780673/CqiWzrma_400x400.jpg",
+  //     },
+  //   },
+  //   {
+  //     post_datetime: adj_date(10).toString(),
+  //     contact: {
+  //       twitter_handle: "mytwitter",
+  //       twitter_url: "google.com",
+  //     },
+  //     name: "bob",
+  //     twitter_photo_url:
+  //       "https://pbs.twimg.com/profile_images/1387824030602780673/CqiWzrma_400x400.jpg",
+  //     description:
+  //       "A description. Lorem ipsum dolor amit. I like pizza, pizza is good to eat. Please givem e more pizza",
+  //     lease_preference: "A day",
+  //     lease_timing: "soon",
+  //     referrer_info: {
+  //       name: "Hello",
+  //       twitter_url: "google.com",
+  //       twitter_photo_url:
+  //         "https://pbs.twimg.com/profile_images/1387824030602780673/CqiWzrma_400x400.jpg",
+  //     },
+  //   },
+  //   {
+  //     post_datetime: adj_date(3).toString(),
+  //     contact: {
+  //       twitter_handle: "mytwitter",
+  //       twitter_url: "google.com",
+  //     },
+  //     name: "bob",
+  //     twitter_photo_url:
+  //       "https://pbs.twimg.com/profile_images/1387824030602780673/CqiWzrma_400x400.jpg",
+  //     description:
+  //       "A description. Lorem ipsum dolor amit. I like pizza, pizza is good to eat. Please givem e more pizza",
+  //     lease_preference: "A day",
+  //     lease_timing: "soon",
+  //     referrer_info: {
+  //       name: "Hello",
+  //       twitter_url: "google.com",
+  //       twitter_photo_url:
+  //         "https://pbs.twimg.com/profile_images/1387824030602780673/CqiWzrma_400x400.jpg",
+  //     },
+  //   },
+  //   {
+  //     post_datetime: "2000-01-10",
+  //     contact: {
+  //       twitter_handle: "mytwitter",
+  //       twitter_url: "google.com",
+  //     },
+  //     name: "bob",
+  //     twitter_photo_url:
+  //       "https://pbs.twimg.com/profile_images/1387824030602780673/CqiWzrma_400x400.jpg",
+  //     description:
+  //       "A description. Lorem ipsum dolor amit. I like pizza, pizza is good to eat. Please givem e more pizza",
```

Comment:
i would delete all of the commented-out code - we can always get older versions of it through the git history